docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,62 @@
+# Resetting your password or reclaiming your username
+
+Whether you have to reset a forgotten password or remember the usernames associated with your {{brand}} account, you can do so via the {{gui}}.
+
+## Reset lost password
+
+Begin by navigating to the {{gui}}.
+At the bottom left-hand side of the page, click on the "Forgot your password?" button.
+
+![Forgot your password button](assets/Forgot-your-password.png)
+
+A pop-up window titled "Reclaim lost password!" appears.
+Type in your username, then click on the "Reclaim password!" button.
+
+![Reclaim password button](assets/Reclaim-lost-password.png)
+
+The content of the pop-up window changes, informing you that an email has been sent to you with instructions on how to reset your password.
+
+Click the "Ok!" button, then go ahead and check your inbox (or spam/junk folder) for an email with the subject **_Cleura Account password reset request_**.
+
+![Reset your password email has been sent](assets/OK-reclaim-lost-password.png)
+
+The email will look like the one shown below.
+Just click the link to create a new password.
+
+![Reset your password email](assets/Reset-password-link.png)
+
+After clicking the link you are landed on a new page, where you are asked to set a new password.
+
+![Type in new password](assets/Choose-new-password.png)
+
+Come up with a strong password and type it in twice, for security reasons.
+Click the "Set new password" button when you are done.
+You will be informed that the password has been reset, so click the "Go back to login page" button to login.
+
+![Password succesfully changed](assets/Reset-completed.png)
+
+## Reclaim your username
+
+Navigate to the {{gui}}, and at the bottom left-hand side of the page click the "Forgot your password?" button.
+
+![Reclaim lost password pop-up](assets/Forgot-your-password.png)
+
+A pop-up window titled "Reclaim lost password!" appears.
+Ignore the "Reclaim password!" button, and follow the link "I don't know my username" instead.
+
+![Forgotten username link](assets/Reclaim-my-username.png)
+
+The pop-up window title changes to "Reclaim lost username".
+Type in your account email address, then click the "Reclaim username" button.
+
+![Reclaim username button](assets/Reclaim-lost-username.png)
```

Comment:
In this screenshot, let's please use an address like `lohcieghoogh1gee@example.com`.